docs/data/registry.json

```diff
@@ -944,7 +941,10 @@
         "6.0",
         "5.0"
       ],
-      "deprecated": ["3.2"]
+      "deprecated": [
+        "5.0",
+        "3.2"
+      ]
```

Comment:
:lady_beetle:
Dedicated Generation 3 hasn't changed, whereas it should have the same split as grid.